src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier color = new ColorSupplier();
+        int number = new Random().nextInt(99) + 1;
```

Comment:
and add a new constant MAX_VALUE = 100
```suggestion
        int number = new Random().nextInt(MAX_VALUE);
```